### DISCUSSION TITLE
hide parameters generated by vuepress-plugin-seo2

### DISCUSSION THREAD

**Discussion by rockbenben** [vuepress-theme-hope/vuepress-theme-hope#2089] [Q&A] [answered]
Accoding to <https://vuepress-theme-hope.github.io/v2/seo/guide.html#ogp>, <https://vuepress-theme-hope.github.io/v2/seo/config.html#ogp> and <https://github.com/vuepress-theme-hope/vuepress-theme-hope/blob/main/demo/seo2/src/.vuepress/config.ts>

I try to hide `og:updated_time` and `article:modified_time` generated by vuepress-plugin-seo2.

```ts
import { seoPlugin } from "vuepress-plugin-seo2";

export default defineUserConfig({
  ...

  plugins: [
    seoPlugin({
      hostname: "https://vuepress-theme-hope.github.io",
      
      updated_time: "2022-07-27T08:00:09.000Z",
      modified_time: "2022-07-27T08:00:09.000Z",
    }),
  ],
});
```
Naturally, this stupid setup won't get any results. 

Could you provide an example of how we can hide one of the parameters generated by vuepress-plugin-seo2?

**Comment by Mister-Hope** [comment, marked as answer]
Drop infomation you don’t want in functions , in the following function, you can override some properties or delete them when returning info.

![EAF7421C-EFF1-4D1D-ABAB-3CC5689DF71D](https://user-images.githubusercontent.com/33315834/181244446-a99fa9fd-f42f-426e-9a52-10a45043aade.png)

**Reply by Mister-Hope** [reply]
Also I do think that docs description are clear enough.

**Reply by rockbenben** [reply]
get it，Thank you very much！

```ts
import { seoPlugin } from "vuepress-plugin-seo2";
export default defineUserConfig({
  ...

  plugins: [
    seoPlugin({
      hostname: "https://vuepress-theme-hope.github.io",
      ogp: (ogp, page) => ({
        ...ogp,
        "og:updated_time": "",
        "og:modified_time": "",
      }),
  ],
});
```
This theme is awesome and beautiful. I transferred all my notes from docsify to VuePress.
